app/elements/pw-shell.html

```diff
@@ -458,10 +462,41 @@
       } else if (response.items) {
         section = resultsElement.querySelector('.yes-results');
 
+        // Clean up the version bit for easier using.
+        for (var item = 0; item < response.items.length; item++) {
+          response.items[item].label = response.items[item].labels[0].displayName;
+        }
+
         // Populate the results in the dom-repeat.
         section.querySelector('template').items = response.items;
 
-        var queryPrefix = '?q=' + this._searchParams.q + '&start=';
+        // Restricted search buttons. Ugh, so. we use the (undocumented) +more:version
+        // search parameter, which means we need to uhhh remove the +more
+        // keyword by hand if it already exists.
+        var matches = this._searchParams.q.match(/(.*) more:(1\.0|2\.0|Blog)/);
+        var searchParams;
+        var activeLabel;
+        if (matches && matches.length > 2) {
+          searchParams = matches[1];
+          activeLabel = matches[2]
```

Comment:
oshit good find, thanks (mine isn't, but i also don't update my jshint out of fear)